### PR TITLE
Adding UUID serviceId defaulting

### PR DESCRIPTION
**Pull request by cschroed-usgs**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>gov.usgs.cida</groupId>
     <artifactId>consul-registrator-maven-plugin</artifactId>
-    <version>1.0.2-SNAPSHOT</version>
+    <version>2.0.0-SNAPSHOT</version>
```
Comment (isuftin) [inline pom.xml]: Does this break API backwards compat? Are you doing server or arbitrary?

Comment (cschroed-usgs) [inline pom.xml]: Yes it does. Semver.

Comment (cschroed-usgs) [inline pom.xml]: *parameter "serviceId" was renamed to "customServiceId"

Comment (isuftin) [inline pom.xml]: Did you want to tag 1.0.2 or set up a Jenkins release job for this?

Comment (cschroed-usgs) [inline pom.xml]: Not really, no functionality was added.

**Pull request merged**